content/build/ci/github-actions/cache.md

```diff
@@ -137,83 +137,74 @@ jobs:
 
 BuildKit doesn't preserve cache mounts in the GitHub Actions cache by default.
 If you wish to put your cache mounts into GitHub Actions cache and reuse it
-between builds, you can use a workaround provided by
+between builds, you can use this action provided by
 [`reproducible-containers/buildkit-cache-dance`](https://github.com/reproducible-containers/buildkit-cache-dance).
 
 This GitHub Action creates temporary containers to extract and inject the
 cache mount data with your Docker build steps.
 
-The following example shows how to use this workaround with a Go project.
```

Comment:
Why drop Go?

Reply:
I am updating the example to match our official example.

Reply:
Why not retain the Go example too?

Reply:
Because the apt package example applies to everyone who uses Ubuntu/Debian regardless of the language and covers the case of multiple paths. 
Also, we don't have or test the go example in [the repo itself](https://github.com/reproducible-containers/buildkit-cache-dance). Also, I think